src/store/modules/config.js

```diff
@@ -0,0 +1,46 @@
+import Vue from "vue";
+export const config = {
+  namespaced: true,
+  state: () => ({
+    ai: {
+      enabled: false,
+    },
+    checklist: {
+      presession: {
+        status: false,
+        tasks: [],
+      },
+      postsession: {
+        status: false,
+        tasks: [],
+      },
+    },
+  }),
```

Comment:
What about all of the other potential state configs?  Like `hotkeys`, `localOnly`, `theme`, etc?

Reply:
yes, will need to add them here for initial state, will do that